extensions/vscode/src/granite/utils/compatibilityUtils.ts

```diff
@@ -2,7 +2,7 @@ import * as vscode from "vscode";
 
 import type { VsCodeWebviewProtocol } from "../../webviewProtocol";
 
-const incompatibleExtensionIds = new Set(["Continue.continue"]);
+const incompatibleExtensionIds = new Set(["redhat.granitecode"]);
 export function checkForIncompatibleExtensions(): boolean {
   const extensions = vscode.extensions.all;
```

Comment:
//remove the id of the currently running extension
incompatibleExtensionIds.delete(context.extension.id);